### PR TITLE
Update overwriting-the-messages-app.md

### PR DESCRIPTION
**Pull request by carolinamenezes**
Update queries and steps
Addition of a new instruction gif

### CODE REVIEW

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 1]
```diff
@@ -1,89 +1,94 @@
...
-6. **According to your store's desired scenario** (catalog or app messages translation), type the following variables into the text box:
+5. Then, click on  **Query Variables** at the bottom of the page. Now, your screen may look like the following:
 
-### Catalog translations
+![queryvariables](https://user-images.githubusercontent.com/60782333/85610649-8e92f280-b62d-11ea-9a5e-aa7ced1a1549.png)
 
-<div class="alert alert-info">
-Use the following variable if you aim to translate text messages from your store's catalog, such as a product name and description.
-</div>
+6. To fill in the "Query Variables" box, check the next sections **according to your store's desired scenario** (*catalog* or *app* messages translations).
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:32]:
```suggestion
6. To fill in the `Query Variables` box, check the next sections **according to your store's desired scenario** (*catalog* or *app* messages translations);
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 2]
```diff
@@ -1,89 +1,94 @@
...
+7. After adjusting your query, click on the play button to run the declared mutation. For both scenarios, the expected response is as follows:
 
+```json
+{
+  "data": {
+    "saveV2": true
+  }
+}
+```
+8. (Optional) If you want to check your changes on the GraphQL IDE, check the " Checking your changes" section.
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:42]: If it's optional, I wouldn't add it as a step.

Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:42]:
```suggestion
Once you receive the expected response, no further actions are needed on your part: you are ready to check out the desired changes in your store!

You also can confirm whether Messages properly saved the new translations or not by performing a query. To know further, check out the **Checking your changes** section.
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 3]
```diff
@@ -1,89 +1,94 @@
...
+
+Now, no further actions are needed on your part. Once you receive the expected response, you are ready to check out the desired changes in your store!
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:44]:
```suggestion
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 4]
```diff
@@ -1,89 +1,94 @@
...
 # Overwriting the Messages app
 
 ## Introduction
 
 Messages is VTEX IO's key tool for your store's **internationalization**, since it's responsible for translating any website message (that is, website text content) for rendering. 
 
-However, you may feel that some translations done by the app are not what you want them to be and therefore wish to change the translated content to something more specific or representative of your store, such as a special login message for Spanish speaking users from Argentina.
+However, you may feel that some translations done by the app are not what you want them to be. Therefore, you may want to change the translated content to something more specific or representative of your store, such as a special login message for Spanish speaking users from Argentina.
 
 This recipe will show you how to overwrite the Messages app and easily create new message content through your store's admin.
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:9]: We can specify here which translations are allowed (messages exported by an app and catalog).

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 5]
```diff
@@ -1,89 +1,94 @@
...
 
-1. [Install](https://vtex.io/docs/recipes/store/installing-an-app) the `vtex.admin-graphql-ide@3.x` app using your terminal;
+1. [Install](https://vtex.io/docs/recipes/store/installing-an-app) the `vtex.admin-graphql-ide@3.x` app using your terminal.
 
 2. Access the **GraphQL admin IDE** section of the desired account. You may find it in the admin's side-bar menu:
 
 ![overwriting-messages-adminsidebarmenu](https://user-images.githubusercontent.com/52087100/66516950-95d29a00-eab8-11e9-8cea-080fbdab84d5.png)
 
-3. From the dropdown list, choose the `vtex.messages` app;
+3. From the dropdown list, choose the `vtex.messages` app.
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:19]: Do you prefer the `.`?

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 6]
```diff
@@ -93,25 +98,128 @@ Use the following variable if you aim to translate text messages exported from a
...
+1. In the **GraphQL admin IDE**, after choosing the `vtex.messages` app, write the following query command in the text box that is displayed:
+
+```json
+query GetTranslation($args2: TranslateArgs!) {
+  translate(args: $args2) 
+} 
+```
+
+2. Then, click on  **Query Variables** at the bottom of the page.
+
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:126]:
```suggestion
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 7]
```diff
@@ -93,25 +98,128 @@ Use the following variable if you aim to translate text messages exported from a
...
+3. To fill in the "Query Variables" box, you must provide the following parameters:
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:127]:
```suggestion
3. To fill in the `Query Variables` box, you must provide the following parameters:
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 8]
```diff
@@ -93,25 +98,128 @@ Use the following variable if you aim to translate text messages exported from a
...
+
+```json
+{
+  "args2": {
+    "indexedByFrom": [
+      {
+      	"from": "en-US",
+      	"messages": [
+          {
+            "content": "Original name of the product in English",
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:145]:
```suggestion
            "content": "Original product name in English",
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 9]
```diff
@@ -93,25 +98,128 @@ Use the following variable if you aim to translate text messages exported from a
...
+1. In the **GraphQL admin IDE**, after choosing the `vtex.messages` app, write the following query command in the text box that is displayed:
+
+```json
+query GetTranslation($args: TranslateWithDependenciesArgs!) {
+  translateWithDependencies(args: $args)
+} 
+```
+
+2. Then, click on  **Query Variables** at the bottom of the page.
+
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:181]:
```suggestion
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 10]
```diff
@@ -93,25 +98,128 @@ Use the following variable if you aim to translate text messages exported from a
...
+3. To fill in the "Query Variables" box, you must provide the following parameters:
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:182]:
```suggestion
3. To fill in the `Query Variables` box, you must provide the following parameters:
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 11]
```diff
@@ -93,25 +98,128 @@ Use the following variable if you aim to translate text messages exported from a
...
+
+- `from`: source message locale.
+- `messages`: a list of the messages you want to check translations, containing the following parameters:
+  - `content`: the `id` of your message string declared in the app's `messages` folder.
+  - `context`: the name of the app in which the message has been overwritten.
+- `to`: target translation locale.
+- `depTree`: the dependency tree as in `"[{\"id\": \"{context}\"}]"`.
```
Comment (guerreirobeatriz) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:189]: I didn't understand what this is about ):

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 12]
```diff
@@ -1,89 +1,95 @@
...
-6. **According to your store's desired scenario** (catalog or app messages translation), type the following variables into the text box:
+5. Then, click on  **Query Variables** at the bottom of the page. Now, your screen may look like the following:
 
-### Catalog translations
+![queryvariables](https://user-images.githubusercontent.com/60782333/85610649-8e92f280-b62d-11ea-9a5e-aa7ced1a1549.png)
 
-<div class="alert alert-info">
-Use the following variable if you aim to translate text messages from your store's catalog, such as a product name and description.
-</div>
+6. To fill in the `Query Variables` box, check the next sections **according to your store's desired scenario** (*catalog* or *app* messages translations);
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:32]:
```suggestion
6. To fill in the `Query Variables` box, check the next sections **according to your store's desired scenario** (*catalog* or *app* messages translations).
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 13]
```diff
@@ -93,25 +99,126 @@ Use the following variable if you aim to translate text messages exported from a
...
+1. In the **GraphQL admin IDE**, after choosing the `vtex.messages` app, write the following query command in the text box that is displayed:
+
+```json
+query GetTranslation($args: TranslateWithDependenciesArgs!) {
+  translateWithDependencies(args: $args)
+} 
+```
+
+2. Then, click on  **Query Variables** at the bottom of the page.
+3. To fill in the `Query Variables` box, you must provide the following parameters:
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:181]:
```suggestion
2. To fill in the `Query Variables` box, you must provide the following parameters:
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 14]
```diff
@@ -93,25 +99,125 @@ Use the following variable if you aim to translate text messages exported from a
...
+1. In the **GraphQL admin IDE**, after choosing the `vtex.messages` app, write the following query command in the text box that is displayed:
+
+```json
+query GetTranslation($args2: TranslateArgs!) {
+  translate(args: $args2) 
+} 
+```
+
+2. Then, click on  **Query Variables** at the bottom of the page.
+3. To fill in the `Query Variables` box, you must provide the following parameters:
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:127]:
```suggestion
2. Then, to fill in the `Query Variables` box, you must provide the following parameters:
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 15]
```diff
@@ -93,25 +99,124 @@ Use the following variable if you aim to translate text messages exported from a
...
+      }
+    ],
+    "to": "en-US",
+    "depTree": "[{\"id\": \"vtex.store-components@3.x\"}]"
   }
 }
 ```
 
-Now, no other actions are needed on your part. Once you receive the expected response, you are ready to check out the desired changes in your store!
+4. After adjusting your query, click on the play button to run it. The expected response is the translated message in the target locale.
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:210]:
```suggestion
3. After adjusting your query, click on the play button to run it. The expected response is the translated message in the target locale. 
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 16]
```diff
@@ -93,25 +99,124 @@ Use the following variable if you aim to translate text messages exported from a
...
+          }
+        ]
+      }
+    ],
+    "to": "pt-BR"
+  }
+}
+```
+
+4. After adjusting your query, click on the play button to run it. The expected response is the translated message in the target locale.
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:155]:
```suggestion
3. After adjusting your query, click on the play button to run it. The expected response is the translated message in the target locale. 
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 17]
```diff
@@ -1,89 +1,95 @@
...
----
-
 # Overwriting the Messages app
 
 ## Introduction
 
 Messages is VTEX IO's key tool for your store's **internationalization**, since it's responsible for translating any website message (that is, website text content) for rendering. 
 
-However, you may feel that some translations done by the app are not what you want them to be and therefore wish to change the translated content to something more specific or representative of your store, such as a special login message for Spanish speaking users from Argentina.
+However, you may feel that some translations done by the app are not what you want them to be. Therefore, you may want to change the translated content to something more specific or representative of your store, such as a special login message for Spanish speaking users from Argentina.
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:7]:
```suggestion
However, you may feel that some translations done by the app are not what you want them to be. Therefore, you may want to change the translated content to something more specific or representative of your store, such as a special login message for Spanish speaking users from Argentina. 

In the next section, we present you a step by step on how to translate a message from your store's catalog, such as a product name or a product description, and text messages exported from an app.
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 18]
```diff
@@ -1,89 +1,97 @@
...
 ## Introduction
 
 Messages is VTEX IO's key tool for your store's **internationalization**, since it's responsible for translating any website message (that is, website text content) for rendering. 
 
-However, you may feel that some translations done by the app are not what you want them to be and therefore wish to change the translated content to something more specific or representative of your store, such as a special login message for Spanish speaking users from Argentina.
+However, you may feel that some translations done by the app are not what you want them to be. Therefore, you may want to change the translated content to something more specific or representative of your store, such as a special login message for Spanish speaking users from Argentina. 
+
+In the next section, we present you a step by step on how to translate a message from your store's catalog, such as a product name or a product description, and text messages exported from an app.
 
 This recipe will show you how to overwrite the Messages app and easily create new message content through your store's admin.
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:11]:
```suggestion
In the next section, we present you a step by step on how to overwrite a translation from your store's catalog, such as a product name or a product description, and text messages exported from an app.
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 19]
```diff
@@ -1,89 +1,95 @@
...
+        "context": "543123",
+        "targetMessage": "Nome do produto em português"
       }
     ]
   }
 }
 ```
+To better understand the full process of overwriting a product message translation, check the following gif:
+
+![ProductTranslation](https://user-images.githubusercontent.com/60782333/85765724-c2811d00-b6ec-11ea-8ada-a9e688c36b86.gif)
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:68]:
```suggestion
```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 20]
```diff
@@ -63,27 +67,26 @@ Use the following variable if you aim to translate text messages from your store
...
 - `to`: target translation locale.
-- `srcLang`: source message locale.
-- `srcMessage`: source message string.
-- `targetMessage`: message translation string.
-- `context`: message translation context. This variable is not mandatory and only serves to give Messages the context desired for the translation, since the same word can have different meaning depending on the language. If you want o use this variable, you'll have to add, between 3 parenthesis, the desired context to the product name, in the admin's catalog. I.e:  `Mouse (((rodent)))`. The desired value will therefore not be displayed next to the product name and will only be used to the catalog translation.
+- `messages`: a list of the messages you want to translate, containing the following parameters:
+    - `srcLang`: source message locale.
+    - `srcMessage`: source message string.
+    - `context`: ID of the product/brand/category that you want to translate. IDs can be found in your store's registration on the admin under Product > Catalog.
+    - `targetMessage`: translated message string.
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:75]:
```suggestion

To better understand the full process of overwriting a product message translation, check the following gif:

![ProductTranslation](https://user-images.githubusercontent.com/60782333/85765724-c2811d00-b6ec-11ea-8ada-a9e688c36b86.gif)

```

File: docs/en/Recipes/development/overwriting-the-messages-app.md [hunk 21]
```diff
@@ -1,59 +1,63 @@
...
+  "saveArgs": {
+    "to": "pt-BR",
     "messages": [
       {
-        "srcLang": "pt-BR",
-        "srcMessage": "Bem vindo! Faça seu Login.",
-        "targetMessage": "Welcome! Please, log in. ",
-        "context": "vtex.login@2.x"
+        "srcLang": "en-US",
+        "srcMessage": "Original name of the product in English",
```
Comment (carolinamenezes) [inline docs/en/Recipes/development/overwriting-the-messages-app.md:58]:
```suggestion
        "srcMessage": "Original product name in English",
```

**Pull request merged**